cardano-cli/src/Cardano/CLI/EraBased/Run/Genesis.hs

```diff
@@ -404,28 +396,28 @@ runGenesisCreateCardanoCmd
     let
       byronGenesis =
         byronGenesis'
-          { gdProtocolParameters =
-              (gdProtocolParameters byronGenesis')
-                { ppSlotDuration = floor (toRational slotLength * recip slotCoeff)
+          { L.gdProtocolParameters =
```

Comment:
So everything Byron related should be exposed from `Cardano.Api.Byron`. Everything else looks good in this PR :+1: